docs/content/workflows/pipelines/actions/plugins/write-plugin.md

```diff
@@ -10,10 +10,16 @@ A step is :
 
 * a builtin action, as GitClone, etc... [read more]({{< relref "workflows/pipelines/actions/builtin/_index.md" >}})
 * a user action [read more]({{< relref "workflows/pipelines/actions/user-actions/_index.md" >}})
-* a Plugin Action
+* a plugin action
 
-A Plugin is a Golang Binary.
+A Plugin is simply an executable which expose a GRPC server corresponding to the right [proto file](https://github.com/ovh/cds/tree/master/sdk/grpcplugin/actionplugin/actionplugin.proto). You can use any languages. The CDS worker will simply query in GRPC the plugin (which is the GRPC server).
 
-Take a look at https://github.com/ovh/cds/tree/master/sdk/plugin/dummy/dummy_plugin.go
+So a CDS plugin have these requirements in order to communicate with worker:
```

Comment:
In order to communicate with a CDS worker, a plugin MUST fill the following requirements: